configure

```diff
@@ -201,14 +200,25 @@ if [ ${nowrf} -eq 0 ]; then
    # the WRF I/O library from the code in $WRF_DIR
    #
    if [ -z "$WRF_DIR" ]; then
-   #   for d in WRF WRF-4.0.3 WRF-4.0.2 WRF-4.0.1 WRF-4.0 WRFV3; do
-      for d in ${standard_wrf_dirs}; do
-         if [ -e ../${d}/external/io_netcdf/libwrfio_nf.a ]; then
-            echo "Found what looks like a valid WRF I/O library in ../${d}"
-            wrf_dir="../${d}"
+      # find WRF dir
+      if [ -e "../WRF/external/io_netcdf/libwrfio_nf.a" ]; then
+        echo "Found what looks like a valid WRF I/O library in ../WRF"
+        WRF_DIR="../WRF"
```

Comment:
Here, as well as on lines 212 and 219, I think `WRF_DIR` should be `wrf_dir`; otherwise, the check on line 235 will fail.